docs/RFCs/00000000_template.md

```diff
@@ -0,0 +1,65 @@
+- Proposal Name:
+- Status: draft/in-progress/completed/rejected/obsolete/postponed
+- Start Date: YYYY-MM-DD
+- Authors:
+- RFC PR: (PR # after acceptance of initial draft)
```

Comment:
Maybe `s/RFC PR/RFC repo and PR`? Or something to indicate which repo the PR is on/in general what repos are involved.

Reply:
Updated the note to say "PR link" instead of "PR #" since the link will also include the repo. Good catch!